spec/app/models/mdm/web_page_spec.rb

```diff
@@ -1,8 +1,61 @@
+require 'webrick'
 RSpec.describe Mdm::WebPage, type: :model do
   it_should_behave_like 'Metasploit::Concern.run'
 
   context 'associations' do
     it { is_expected.to belong_to(:web_site).class_name('Mdm::WebSite') }
+
+    context 'serialized cookie attribute' do
```

Comment:
`#cookie` is not an association, so it should not be under the `'associations'` context.  Move `context 'serialized cookie attribute'` to be a sibling of `context 'associations'` and call it `context '#cookie'` or do

```ruby
context 'serialized attributes' do
  context 'cookie' do
    # stuff currently in `context 'serialized cookie attribute'`
  end
end
```